app.grayjay.Desktop.yaml

```diff
@@ -0,0 +1,85 @@
+app-id: app.grayjay.Desktop
```

Comment:
`Desktop` is NOT an acceptable application name.

Reply:
See https://docs.flathub.org/docs/for-app-authors/requirements#application-id

Reply:
This has been addressed. Apologies for the oversight